3761/1/gem_statistic_analyzer.rb

```diff
@@ -0,0 +1,12 @@
+class GemStatisticAnalyzer
+  RATIO = { used_by: 0.9, watch: 0.8, star: 1, fork: 0.85,
+            contributors: 0.75, issues: 0.75 }.freeze
+
+  def self.call(stats)
+    stats[:score] = stats.inject(0) do |score, parametr|
+      score += parametr[1] * RATIO[parametr[0]] if parametr[1].is_a? Integer
```

Comment:
again what these magic digits mean? I don't understand

Reply:
It is coefficient to calculate a score